src/components/Calendar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Field } from 'redux-form';
+import { DatePicker } from 'redux-form-material-ui';
```

Comment:
It just means more work when we do migrate away from material-ui. But maybe we should stay with material-ui for now.